src/main/java/com/o19s/es/ltr/ranker/normalizer/Normalizers.java

```diff
@@ -0,0 +1,55 @@
+package com.o19s.es.ltr.ranker.normalizer;
+
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * Class that manages Normalizer implementations
+ */
+public class Normalizers {
+    private static final Map<String, Normalizer> NORMALIZERS = new HashMap<>();
+    public static final String DEFAULT_NORMALIZER_NAME = "noop";
+
+    static {
+        register("noop", new NoopNormalizer());
+        register("sigmoid", new SigmoidNormalizer());
+    }
+
+    public static void register(String name, Normalizer normalizer) {
+        NORMALIZERS.put(name, normalizer);
+    }
+
+    public static Normalizer get(String name) {
+        Normalizer normalizer = NORMALIZERS.get(name);
+        if (normalizer == null) {
+            throw new IllegalArgumentException(name + " is not a valid Normalizer");
+        }
+        return normalizer;
+    }
+
+    public static boolean exists(String name) {
+        return NORMALIZERS.containsKey(name);
+    }
+
+    public static Normalizer defaultNormalizer() {
```

Comment:
nit: perhaps name it noopNormalizer() ?
having a defaultNormalizer() method here feels like it can be changed while in fact it does not seem to be the responsibility of this class to change the "default" normalizer of the rankers using a Normalizer.

Reply:
I'm thinking maybe just get rid of the default method. The noop normalizer kind of speaks for itself.